cockatrice/src/userinfobox.cpp

```diff
@@ -26,6 +28,8 @@ UserInfoBox::UserInfoBox(AbstractClient *_client, bool _fullInfo, QWidget *paren
     userLevelLabel1 = new QLabel;
     userLevelLabel2 = new QLabel;
     userLevelLabel3 = new QLabel;
+    accountAgeLebel1 = new QLabel;
+    accountAgeLabel2 = new QLabel;
```

Comment:
Can you convert the `new` calls to stack allocation instead? https://github.com/Cockatrice/Cockatrice/blob/master/CONTRIBUTING.md#memory-management